### PR TITLE
<PlayerHandSelectors>

### PR DESCRIPTION
**Pull request by ryotogashi**

### CODE REVIEW

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: pageComponents/NewPostPage/PlayerHandSelectors.tsx [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import * as React from "react";
+import styled from "styled-components";
+import SelectablePlayingCard from "../../components/SelectablePlayingCard";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PlayingCard, { Rank, Suit } from "../../models/PlayingCard";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  playerIndexes: number[];
+  playerHands: { left: PlayingCard | null; right: PlayingCard | null }[];
```
Comment (axross) [inline pageComponents/NewPostPage/PlayerHandSelectors.tsx:10]: I think we can use tuple for this kind of situation

```suggestion
  playerHands: [PlayingCard | null, PlayingCard | null][];
```

**Comment by ryotogashi** [comment]
@axross Can you check when you have time?

File: pageComponents/NewPostPage/PlayerHandSelectors.tsx [hunk 2]
```diff
@@ -0,0 +1,117 @@
+import * as React from "react";
+import styled from "styled-components";
+import SelectablePlayingCard from "../../components/SelectablePlayingCard";
+import { MOBILE_MEDIA } from "../../constants/mediaquery";
+import PlayingCard, { Rank, Suit } from "../../models/PlayingCard";
+import getPositionByPlayerAndIndex from "../../utilities/getPositionByPlayerAndIndex";
+
+interface Props extends React.Attributes {
+  players: {
+    index: number;
```
Comment (axross) [inline pageComponents/NewPostPage/PlayerHandSelectors.tsx:10]: I'm sorry, let me make sure. What does `index` mean? Are `players[i]` and `index` different?

File: pageComponents/NewPostPage/PlayerHandSelectors.tsx [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+    hand: [PlayingCard | null, PlayingCard | null];
+  }[];
+  onChange?: (index: number, hand: Hand, rank: Rank, suit: Suit) => void;
```
Comment (axross) [inline pageComponents/NewPostPage/PlayerHandSelectors.tsx:13]: I think we should use prop names and signatures `players` and `onChange`, as like `value` and `onChange` of standard `<input>`. And let's consider `value` is actually `value` or should be `defaultValue`.

File: pageComponents/NewPostPage/PlayerHandSelectors.tsx (hunk 2 above)
Comment (ryotogashi) [reply pageComponents/NewPostPage/PlayerHandSelectors.tsx:10]: It is a PlayerIndex the is still active at the Showdown phase.

Comment (axross) [reply pageComponents/NewPostPage/PlayerHandSelectors.tsx:10]: How are they different?

**Pull request closed without merging**